### DISCUSSION TITLE
Immich with several user accounts

### DISCUSSION THREAD

**Discussion by temium62** [simulot/immich-go#292] [Q&A] [answered]
Hello
I just uploaded my Google Photo account to Immich and everything is ok. 
I wanted to know if it was possible to upload with other Google accounts to reciprocal Immich accounts?
Does the Key automatically direct to the correct profile?
Thanks in advance.

**Comment by simulot** [comment, marked as answer]
Create a key per user account.
Use the user's key to upload in the user's account.

**Reply by temium62** [reply]
Thank you .
Je préférais demander par sécurité avant de faire la manip. 
Merci encore pour cet outil hyper pratique  !
